lib/plausible_web/components/generic.ex

```diff
@@ -58,28 +73,31 @@ defmodule PlausibleWeb.Components.Generic do
     """
   end
 
-  attr(:title, :string, default: "Notice")
+  attr(:title, :any, default: nil)
   attr(:size, :atom, default: :sm)
+  attr(:theme, :atom, default: :yellow)
   attr(:dismissable_id, :any, default: nil)
   attr(:class, :string, default: "")
   attr(:rest, :global)
   slot(:inner_block)
 
   def notice(assigns) do
+    assigns = assign(assigns, :theme, Map.fetch!(@notice_themes, assigns.theme))
+
     ~H"""
     <div id={@dismissable_id} class={@dismissable_id && "hidden"}>
-      <div class={"rounded-md bg-yellow-50 dark:bg-yellow-100 p-4 relative #{@class}"} {@rest}>
+      <div class={"rounded-md #{@theme.bg} p-4 relative #{@class}"} {@rest}>
```

Comment:
Minor suggestion: by using an array, it avoids that extra space in the element class list when a custom class is not present, e.g. `rounded-md p-4 relative `

```suggestion
      <div class={["rounded-md p-4 relative", @theme.bg, @class]} {@rest}>
```